spec/classes/init_spec.rb

```diff
@@ -88,7 +88,7 @@
         it { is_expected.to contain_file('/usr/local/etc/poudriere.conf').with(content: %r{^POUDRIERE_DATA=/data$}) }
         it { is_expected.to contain_file('/usr/local/etc/poudriere.conf').with(content: %r{^USE_PORTLINT=yes$}) }
         it { is_expected.to contain_file('/usr/local/etc/poudriere.conf').with(content: %r{^MFSSIZE=4G$}) }
-        it { is_expected.to contain_file('/usr/local/etc/poudriere.conf').with(content: %r{^USE_TMPFS=data\\ wrkdir$}) }
+        it { is_expected.to contain_file('/usr/local/etc/poudriere.conf').with(content: %r{^USE_TMPFS="data wrkdir"$}) }
```

Comment:
Ruby `shellescape()` vs. Puppet `shellquote()` leads to minor differences in the test suite.